content/post/102-gossipsub-reports.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2020-07-06
+url: /gossipsubv1.1-eval-report-and-security-audit/
+title: A comprehensive Evaluation of GossipSub-v1.1 (and a new Logo!)
+header_image: 102-header-image-gossipsub-reports.png
+author: David Dias
+tags: go-libp2p, gossipsub
+---
+
+<p align="center">
+  <img src="https://gateway.ipfs.io/ipfs/QmVSk8VvxWExnYCjqg7TNW2aBnN8Wr7SzkawLUyocWq96p/Gossipsub_fullcolor.png" alt="Gossipsub logo" width="400" />
+</p>
+
+We are back with a direct follow up on the [Gossipsub v1.1 release from May](https://blog.ipfs.io/2020-05-20-gossipsub-v1.1) with the much awaited Evaluation Report and three other presents we have for you.
+
+## ✨ The Logo
+
+The first cat was out of the bag as soon as you opened this post – the new logo! We now have a dedicated logo for this libp2p PubSub router implementation which represents the double network nature of Gossipsub. We hope you enjoy it!
+
+You can find all the [logo assets here](https://ipfs.io/ipfs/QmVSk8VvxWExnYCjqg7TNW2aBnN8Wr7SzkawLUyocWq96p)
+
+## 📊 Evaluation Report
+
+<p align="center">
+  <a href="https://gateway.ipfs.io/ipfs/QmWR376YyuyLewZDzaTHXGZr7quL5LB13HRFnNdSJ3CyXu/Gossipsub%20Evaluation%20Report.pdf"><img src="https://gateway.ipfs.io/ipfs/QmWR376YyuyLewZDzaTHXGZr7quL5LB13HRFnNdSJ3CyXu/report-eval.png" width="600" /></a>
+</p>
+
+We are sharing with you a comprehensive, 61-page evaluation report, in which you can learn how we approached the testing of Gossipsub v1.1, the setting in which tests were run, and detailed descriptions of the conclusions we were able to take from such evaluation. With this evaluation, we demonstrate that GossipSub is resilient against all of the attacks studied, capable of recovering the mesh and meeting the message delivery deadline requirements of the Filecoin and the ETH2.0 blockchains.
+
+In addition to this report, which you can [find here](https://gateway.ipfs.io/ipfs/QmWR376YyuyLewZDzaTHXGZr7quL5LB13HRFnNdSJ3CyXu/Gossipsub%20Evaluation%20Report.pdf), we are also excited to share with you:
+
+- The code that runs the malicious actor, acronymed “bad boy”, is simpler, more lightweight, and faster than the actual GossipSub protocol code. This is because attackers don’t need to follow all protocol logic and benefit from running only the essential functions with the goal of degrading the quality of service of the network. Find the [code here](https://github.com/libp2p/gossipsub-hardening/blob/master/test/badboy.go).
```

Comment:
```suggestion
- The code that runs the malicious actor, code-named  “bad boy”, which is simpler, more lightweight, and faster than the actual GossipSub protocol code. This is because attackers don’t need to follow all protocol logic and benefit from running only the essential functions with the goal of degrading the quality of service of the network. Find the [code here](https://github.com/libp2p/gossipsub-hardening/blob/master/test/badboy.go).
```